package.json

```diff
@@ -66,7 +66,7 @@
   "husky": {
     "hooks": {
       "commit-msg": "d2-style commit check",
-      "pre-commit": "d2-style js apply"
+      "pre-commit": "d2-style js apply && npm run localize && git add ./i18n"
```

Comment:
Will this not fail if the commit is run from a nested folder in the project?

Reply:
Not a huge deal, but you could maybe use this syntax to circumvent it: https://superuser.com/a/550068